src/species/ca40.jl

```diff
@@ -0,0 +1,76 @@
+using .PhysicalConstants:PhysicalConstant
+
+export Ca40
+
+const properties_ca40 = (mass = PhysicalConstant(6.635943757345042e-26, "kg"),
+
+                         charge = 1,
+
+                         nuclearspin = 0,
+
+                         full_level_structure = OrderedDict("S1/2" => (n=4, l=0, j=1//2, f=1//2, E=0),
+                                                            "D3/2" => (n=3, l=2, j=3//2, f=3//2, E=4.09335071228e14),
+                                                            "D5/2" => (n=3, l=2, j=5//2, f=5//2, E=4.1115503183857306e14),
+                                                            "P1/2" => (n=4, l=1, j=1//2, f=1//2, E=7.554e14),
+                                                            "P3/2" => (n=4, l=1, j=3//2, f=3//2, E=7.621e14),
+                                                           ),
+
+                         full_transitions = Dict(("S1/2", "D5/2") => (multipole="E2", einsteinA=8.562e-1),
+                                                 ("S1/2", "P1/2") => (multipole="E1", einsteinA=1.299e8),
+                                                 ("D3/2", "P1/2") => (multipole="E1", einsteinA=1.060e7),
+                                                 ("S1/2", "D3/2") => (multipole="E2", einsteinA=9.259e-1),
+                                                 ("S1/2", "P3/2") => (multipole="E1", einsteinA=1.351e8),
+                                                 ("D3/2", "P3/2") => (multipole="E1", einsteinA=1.110e6),
+                                                 ("D5/2", "P3/2") => (multipole="E1", einsteinA=9.901e6),
+                                                ),
+                        
+                         # Optional fields
+                         default_sublevel_selection = [("S1/2", "all"),
+                                                       ("D5/2", "all"),
+                                                      ],
+
+                         gfactors = Dict("S1/2" => 2.00225664,
+                                         "D5/2" => 1.2003340
+                                        ),
+
+                         #nonlinear_zeeman = Dict(("S1/2", -1//2) => B->1.3e-4*B^2,

```

Comment:
Do we need this?

Reply:
The new code allows for assigning an ion species an optional nonlinear term of the Zeeman shift (relevant in some use cases). We have no need for it in our first Ca40 use case. (or do we?) These commented lines are a crude way for me to remember what the proper syntax of specifying such a nonlinear term would be.